### PR TITLE
spec: interfaceToC: remove duplicate ulong type on the table

### PR DESCRIPTION
**Pull request by ljmf00**
Signed-off-by: Luís Ferreira <contact@lsferreira.net>

---

By reading https://dlang.org/spec/interfaceToC.html, I found out that `ulong` is duplicated and the first one seems wrong/incomplete.

**EDITED:** I also removed extra newlines.

### CODE REVIEW

**Comment by ljmf00** [comment]
I think [this](https://dlang.org/articles/ctod.html#types) article is also wrong. Can someone confirm?

```
long               =>        int
unsigned long      =>        uint
```
This is not entirely right since `long` and `unsigned long` is dependent on the byte size.

**Comment by Geod24** [comment]
> This is not entirely right since long and unsigned long is dependent on the byte size.

It should probably point to the right Druntime definition, too.

**Comment by ljmf00** [comment]
> > This is not entirely right since long and unsigned long is dependent on the byte size.
> 
> It should probably point to the right Druntime definition, too.

Changed

File: articles/ctod.dd [hunk 1]
```diff
@@ -88,8 +88,8 @@ unsigned short     =&gt;        ushort
 wchar_t            =&gt;        wchar
```
Comment (PetarKirov) [inline articles/ctod.dd:88]: @ljmf00 Can you also fix this:

```suggestion
wchar_t            =&gt;        core.stdc.stddef.wchar_t
```

(For reference: https://github.com/dlang/druntime/blob/v2.098.0/src/core/stdc/stddef.d#L27-L36)

Comment (ljmf00) [reply articles/ctod.dd]: I didn't notice it, thanks. Fixed

**Review by PetarKirov — APPROVED** [review]
Thanks!

**Pull request merged**